File: SistemaProcinco/SistemaProcinco.Common/Models/EmpleadosViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SistemaProcinco.Common.Models
{
    public class EmpleadosViewModel
    {
        public int Empl_Id { get; set; }
        public string Empl_DNI { get; set; }
        public int? Carg_Id { get; set; }
        public string Empl_Nombre { get; set; }
        public string Empl_Apellido { get; set; }
        public string Empl_Correo { get; set; }
        public DateTime? Empl_FechaNacimiento { get; set; }
        public string Empl_Sexo { get; set; }
        [NotMapped]
        public string Sexo { get; set; }
        public int? Estc_Id { get; set; }
        [NotMapped]
        public string EstadoCivil { get; set; }
        public string Empl_Direccion { get; set; }
        public string Ciud_id { get; set; }
        [NotMapped]
        public string Ciudades { get; set; }
        public int? Empl_UsuarioCreacion { get; set; }
        public DateTime? Empl_FechaCreacion { get; set; }
        public int? Empl_UsuarioModificacion { get; set; }
        public DateTime? Empl_FechaModificacion { get; set; }
        public decimal? Empl_SalarioHora { get; set; }

        [NotMapped]
        public string Creacion { get; set; }

        [NotMapped]
        public string Modificacion { get; set; }
    }
}

File: PlantillaProcinco/src/app/Models/EmpleadosViewModel.ts
export class Empleado {
    empl_Id !: Number;
    empl_DNI  !: String;
    carg_Id !: Number;
    empl_Nombre !: String;
    empl_Apellido !: String;
    empl_Correo !: String;
    empl_FechaNacimiento !: String;
    empl_Sexo !: String;
    sexo !: String;
    estc_Id !: Number;
    estadoCivil !: String;
    empl_Direccion !: String;
    ciud_id !: String;
    ciudades !: String;
    empl_UsuarioCreacion !: Number;
    empl_FechaCreacion !: String;
    empl_UsuarioModificacion !: Number;
    empl_FechaModificacion !: String;
    empl_SalarioHora !: Number;
    usuarioCreacion!: String;
    usuarioModificacion!: String;
}

export class dropEmpleados{
    value?:String;
    text?:String;
}
